content/en/blog/2024/2024-06-pachli-introduction.md

```diff
@@ -0,0 +1,68 @@
+
+As an open source project contributions of code and documentation through the traditional pull-request route is always welcome. The current [list of open issues](https://github.com/pachli/pachli-android/issues) may provide ideas.
+
+I'm especially interested in hearing from people who are getting started with Android development and want to develop their skills further. I'm very happy to spend time helping people onboard the project, understand the code, and make their first contributions.
+
+There are many other extremely valuable ways you can contribute to an open source project that are often insufficiently recognised, and I want to explicitly call those out (this is not an exhaustive list):
+
+- Translating the UI
+- Improving the user interface / user experience
+	- E.g., identifying user journeys that are needlessly difficult to complete or discover and describing how they could be improved
+- Curating the list of open issues
+	- Following up on incomplete issues
+	- Contacting an issue reporter to see if a fix solved their problem
+	- Making sure issues are not ignored
+	- Applying consistent labels to make issues easier to manage
+- Improving the onboarding experience for new contributors
+- Writing release notes
+- Curating the project's social media account
+	- Converting user problems reports on social media to properly tracked issues
+	- Responding to users who have questions
+	- Identifying patterns of questions that point to where the software can be improved
+- Liaising between the Pachli project and other Fediverse projects
+
+These are often seen as non-technical, "soft" skills that are undervalued. This project will not do that.
+
+The project's [documentation for contributors](https://github.com/pachli/pachli-android/blob/main/CONTRIBUTING.md) has links to more specific documentation for those activities.
+
+Of course, if you see something you think the project should be doing, but is not, and you have the skills to do it, you should volunteer. Explain the problem, how you'd solve it, and get to it.
+
+As the project takes in funding through membership I also want the membership to contribute and vote on proposals for how funding could be spent.
+
+For example (these are not fully fleshed out proposals, just two examples of things we could do):
+
+- Fund an accessibility review of the app, recommending specific changes to improve accessibility
+- Fund an internship through a programme like [Outreachy](https://www.outreachy.org/)
+
+If any of the above is interesting to you then please get in touch:
+
+- Mastodon: [@pachli@mastodon.social](https://mastodon.social/@pachli)
+- GitHub discussions: https://github.com/pachli/pachli-android/discussions
+- E-mail: [team@pachli.app](mailto:team@pachli.app)
```

Comment:
Also somewhere in here there should be some prose similarish to the following:

The Pachli project has \[a CLA\]\(LINK\). In order to contribute code or docs you will need to sign the CLA, which can be done by \[instructions\]\(LINK\).

The Pachli project uses the LICENSE license.

Reply:
Good idea, thanks, and done. I've also mentioned and linked to the code of conduct.